src/main/java/core/basesyntax/model/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.model;
+
+public class Circle extends Color {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area: " + getArea() + " sq. units, radius: "
+                + radius + " units, color: " + getColor());
```

Comment:
Since Circle should not extend Color, 'getColor()' method will not be available. You need to have a Color field and provide a method to get the color of the circle.